src/client/components/DataGrid/DataCell/DataCell.scss

```diff
@@ -134,9 +133,27 @@ $border: 1px solid $ui-border;
     &.lastCol {
       border-left: $border;
     }
+
+    &.highlighted {
+      &.firstCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-right, $highlight-border-shadow;
+        clip-path: inset(-4px -4px -4px 0px); // Adds shadow to top, bottom, right
+      }
+
+      &.lastCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-left, $highlight-border-shadow;
+        clip-path: inset(-4px 0px -4px -4px); // Adds shadow to top, bottom, left
+      }
+
+      &.firstCol.lastCol::before {
+        box-shadow: $highlight-top-bottom, $highlight-left, $highlight-right, $highlight-border-shadow;
+        clip-path: inset(-4px -4px -4px -4px); // Adds shadow to top, bottom, left, right
+      }
+    }
   }
 
-  .data-cell.editable {
+  .data-cell.editable:not(.noBorder) {
+    border-left: unset;
```

Comment:
This fixes the duplicated borders in between rows that looked like this:
 
![image](https://github.com/openforis/fra-platform/assets/41337901/1ac1a870-f753-497c-ab35-048c232468c4)

And now looks like this (how we have it for non-rtl):

![image](https://github.com/openforis/fra-platform/assets/41337901/cc2f9585-e918-4f02-ab08-f1ff3dc02835)